File: 3476-find-minimum-operations-to-make-all-elements-divisible-by-three/find-minimum-operations-to-make-all-elements-divisible-by-three.dart
class Solution {
  int minimumOperations(List<int> nums) {
    int operations = 0;
    
    for (int num in nums) {
      int remainder = num % 3;
      if (remainder == 1) {
        operations += 1;
      } else if (remainder == 2) {
        operations += 1;
      }
    }
    
    return operations;
  }
}

File: 3476-find-minimum-operations-to-make-all-elements-divisible-by-three/find-minimum-operations-to-make-all-elements-divisible-by-three.go
func minimumOperations(nums []int) int {
    operations := 0
    
    for _, num := range nums {
        remainder := num % 3
        if remainder == 1 {
            operations += 1
        } else if remainder == 2 {
            operations += 1
        }
    }
    
    return operations
}
